src/pages/workspace/companyCards/WorkspaceCompanyCardsPage.tsx

```diff
@@ -86,6 +123,8 @@ function WorkspaceCompanyCardPage({route}: WorkspaceCompanyCardPageProps) {
                         <WorkspaceCompanyCardsListHeaderButtons
                             policyID={policyID}
                             selectedFeed={selectedFeed}
+                            shouldShowAssignCardButton={isPending || !isEmptyObject(cardsList)}
```

Comment:
I think its because the `cardList` contains a nested cardList for the custom feeds with the cards available to assign so it its not empty object potentially, I think we need to filter out the `cardList` key and only look at the cards with the cardIDs @dukenv0307 @DylanDylann @VickyStash